### DISCUSSION TITLE
Break the Mentorship Project tie

### DISCUSSION THREAD

**Discussion by aaronreed708** [finos/a11y-theme-builder#815] [Polls]
In https://github.com/finos/a11y-theme-builder/discussions/813 we created a poll to help us determine what the Theme Builder mentorship project should be.  We had a tie between two options.  This poll is to put the two issues up for a tie-breaking vote.  As you vote, please consider:

- You feel that there is a high probability that the work can be contained in the given timeframe (3 months of work full time or 6 months work part time)
- You feel that there is a high likelihood of success for the mentee; we'd like to keep them in the community after the mentorship ends
- The project should be impactful and something that we can proudly point to as a 2024 accomplishment.

**Comment by PaulaPaul** [comment]
If we have a tie here, don't forget we also need some 'challenges' we can offer to attendees at the GAAD event in Chicago - maybe we can tee up some of the work between now and then and use things that are not part of the Mentorship project as 'challenges' for people to work on at that hackathon event!
